downstream/modules/platform/con-controller-execution-nodes.adoc

```diff
@@ -0,0 +1,19 @@
+[id="con-controller-execution-nodes"]
+
+= Execution Nodes
+
+Vertical scaling an execution node provides more forks for job execution. 
+As mentioned in the xref:ref-controller-capacity-planning-exercise[Example capacity planning exercise], a host with 16 GB of memory is, by default, assigned the capacity to run 137 “forks”, which at the default setting of five forks per job, can run around 22 jobs concurrently. 
```

Comment:
should the statement be: which _is_ the default setting of five forks?

Reply:
the default setting _is_ five forks, but this is referring to the 137 forks, 137 divided by the default 5 is ~22. As far I know. (Actually 137/5 ~27, but that's what the original says, perhaps there's some overhead.)